github/repos_test.go

```diff
@@ -42,15 +42,17 @@ func TestRepositoriesService_List_specifiedUser(t *testing.T) {
 		testMethod(t, r, "GET")
 		testHeader(t, r, "Accept", mediaTypeLicensesPreview)
 		testFormValues(t, r, values{
-			"type":      "owner",
-			"sort":      "created",
-			"direction": "asc",
-			"page":      "2",
+			"visibility":  "public",
+			"affiliation": "owner,collaborator",
+			"type":        "owner",
+			"sort":        "created",
+			"direction":   "asc",
+			"page":        "2",
 		})
 		fmt.Fprint(w, `[{"id":1}]`)
 	})
 
-	opt := &RepositoryListOptions{"owner", "created", "asc", ListOptions{Page: 2}}
+	opt := &RepositoryListOptions{"public", "owner,collaborator", "owner", "created", "asc", ListOptions{Page: 2}}
```

Comment:
With this many fields, I think it's time to start using keyed composite literals.